packages/experience/src/apis/experience.ts

```diff
@@ -0,0 +1,72 @@
+import {
+  type IdentificationApiPayload,
+  InteractionEvent,
+  type PasswordVerificationPayload,
+  SignInIdentifier,
+  type UpdateProfileApiPayload,
+} from '@logto/schemas';
+
+import api from './api';
+
+const prefix = '/api/experience';
+
+const experienceRoutes = Object.freeze({
```

Comment:
We have an `experience.routes` constant in the `schema` package, maybe use `experienceApiRoutes` or something.